lib/msal-browser/src/controllers/IController.ts

```diff
@@ -121,4 +121,6 @@ export interface IController {
 
     /** @internal */
     getPerformanceClient(): IPerformanceClient;
+
+    isPlatformBrokerAvailable(): Promise<boolean>;
```

Comment:
This should not be here. It should be 
```
export function isPlatformBrokerAvailable(): Promise<boolean>
```
As a function out of the Controller class.

Reply:
Removed!